pkg/cdi/cache.go

```diff
@@ -254,6 +259,19 @@ func (c *Cache) InjectDevices(ociSpec *oci.Spec, devices ...string) ([]string, e
 	return nil, nil
 }
 
+func (c *Cache) UpdateMounts(ociSpec *oci.Spec, mounts ...string) ([]string, error) {
+	unresolved, edits, _ := c.filterDevices(ociSpec, mounts)
+	if unresolved != nil {
+		fmt.Printf("unresolvable CDI mount %s", strings.Join(mounts, ", "))
```

Comment:
We should not just print to STDOUT here.

Reply:
`UpdateMounts` removed